cmd/clusterctl/cmd/root.go

```diff
@@ -48,12 +54,33 @@ func Execute() {
 func init() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 
-	verbosity := flag.CommandLine.Int("v", 0, "Set the log level verbosity.")
-	logf.SetLogger(logf.NewLogger(logf.WithThreshold(verbosity)))
+	verbosity = flag.CommandLine.Int("v", 0, "Set the log level verbosity. You can set the CLUSTERCTL_LOG_LEVEL environment variable.")
```

Comment:
Nit: I'd write "Alternately, you can set..." to indicate you don't need to do both.  Or possibly "This overrides the CLUSTER_LOG_LEVEL environment variable."

Reply:
like the second :), made the change